docs/tutorials/projects/light-up.md

```diff
@@ -191,13 +191,11 @@ You need to tell the code how to access your specific robot (which in this case
 
    {{% snippet "show-secret.md" %}}
 
-2. In the code sample, find the `payload`, a long string of numbers and letters.
-   Copy it and paste it into line 13 of <file>lightupbot.py</file> in place of `ROBOT_SECRET`.
-3. Find the robot address, of the form `robot-name-main.abc1ab123a1.viam.cloud`, and paste it into line 14 of <file>lightupbot.py</file> in place of `ROBOT_ADDRESS`.
```

Comment:
is it possible a tool came through and accidentally renumbered these? looks like the step numbering got garbled here and below!

Reply:
Good thinking but I manually changed that - If you use 1. within markdown it will automatically use the correct numbers (and when we remove steps we don't have to manually change numebers...)